app/views/users/index.erb

```diff
@@ -1 +1,12 @@
-Here is a list of users
\ No newline at end of file
+<%= stylesheet_link_tag "users" %>
+<% @users.each do |user| %>
+  <div class="user-row">
+    <div class="profile-photo">
+      <div class="placeholder-photo"></div>
+    </div>
+    <div class="user">
+      <h2><%= user.name %></h2>
+      <p>Number of posts: <%= user.posts_counter || 0 %></p>
+    </div>
+  </div>
+<% end %>
```

Comment:
- [ ] Kindly consider clicking on each user takes you to their show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png)